api/src/main/resources/metadata/Role_Privilege.xml

```diff
@@ -783,6 +783,8 @@
     <role_privilege role="Triage" privilege="View Triage Metrics"/>
     <role_privilege role="Triage" privilege="View Triage Queuelist"/>
     <role_privilege role="Triage" privilege="View Triage Menu"/>
+    <role_privilege role="Triage"  privilege="Move Patient To Next Service Point Button" />
```

Comment:
Where is the actual privilege
Can we also make the privilege less of a sentence and more of a privilege? Make precise and concise

Reply:
This is it `https://github.com/METS-Programme/openmrs-module-ugandaemr/blob/7176c0d6192425a9ad79a4b98a6d0ac550d3a140/api/src/main/resources/metadata/Role_Privilege.xml#L577`